backend/src/main/java/org/example/backend/DTO/CreateItemDTO.java

```diff
@@ -0,0 +1,17 @@
+package org.example.backend.DTO;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import org.example.backend.Model.Section;
+import org.example.backend.Model.Unit;
+
+public record CreateItemDTO(
+        @NotBlank @Size(min = 2, max = 100) String name,
+        @Min(1) double quantity,
```

Comment:
I think it would be better for the moment to use a string as quantity (no unit of measurement), to keep things simple.

Reply:
Generally in frontend forms, you would explicitly use `type="number"` for the Quantity input of the form. I wrote a message in Slack for you.